### PR TITLE
[INJIMOB-1850] change push trigger workflow branch name for docker pu…

### PR DESCRIPTION
**Pull request by abhip2565**

### CODE REVIEW

File: .github/workflows/push-trigger.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ on:
       - MOSIP*
       - qa-develop
       - inji-*
+      - injimob-1850
```
Comment (vijay151096) [inline .github/workflows/push-trigger.yml:25]: Lets remove it.

File: .github/workflows/push-trigger.yml [hunk 2]
```diff
@@ -22,6 +22,7 @@ on:
       - MOSIP*
       - qa-develop
       - inji-*
```
Comment (vijay151096) [inline .github/workflows/push-trigger.yml:24]: and update this to inji*

File: .github/workflows/push-trigger.yml (hunk 1 above)
Comment (abhip2565) [reply .github/workflows/push-trigger.yml]: Done

File: .github/workflows/push-trigger.yml (hunk 2 above)
Comment (abhip2565) [reply .github/workflows/push-trigger.yml]: done

**Pull request merged**